demos/showcase/package.json

```
@@ -6,9 +6,10 @@
         "@material-ui/core": "^4.0.0",
         "@material-ui/icons": "^4.0.0",
         "@material-ui/styles": "^4.5.0",
+        "@pxblue/colors": "^3.0.0",
         "@pxblue/icons-mui": "^2.1.0",
         "@pxblue/react-progress-icons": "^2.0.0",
-        "@pxblue/react-themes": "5.0.0-beta.0",
+        "@pxblue/react-themes": "5.0.0",
```

this should be using a caret anyway...new theme coming shortly

```suggestion
        "@pxblue/react-themes": "^5.0.1",
```